revscoring/dependencies/dependent.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, name, process=None, depends_on=None,
         self.calls = 0
 
     def __call__(self, *args, **kwargs):
-        logger.debug("Executing {0}.".format(self))
+        logger.debug("Executing {0} ({1} calls so far)."
+                     .format(self, self.calls))
         self.calls += 1
```

Comment:
Looks good then.
Is it a problem that "All checks have failed" or can this be safely merged?